packages/kit/src/core/prerender/prerender.js

```diff
@@ -410,6 +416,14 @@ export async function prerender() {
 
 	await q.done();
 
+	for (const [route_id, prerender] of prerender_map) {
+		if (prerender === true && !prerendered_routes.has(route_id)) {
+			throw new Error(
+				`Route "${route_id}" is marked as prerenderable, but was not prerendered. See https://kit.svelte.dev/docs/page-options#prerender-troubleshooting for more info`
```

Comment:
Should we wait with throwing this error until we checked all routes? I don't know how often that is, but maybe people could get annoyed seing this trickle in one by one, having to rerun their build each time.
```
The following routes were marked as prerenderable, but were not prerender:
- a
- b
- ..
See ... for more info
```

Reply:
Yeah, that's way better. Updated